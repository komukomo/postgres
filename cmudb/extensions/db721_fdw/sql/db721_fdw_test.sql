CREATE EXTENSION IF NOT EXISTS db721_fdw;
CREATE SERVER IF NOT EXISTS db721_server FOREIGN DATA WRAPPER db721_fdw;
CREATE FOREIGN TABLE IF NOT EXISTS db721_farm
(
    farm_name       varchar,
    min_age_weeks   real,
    max_age_weeks   real
) SERVER db721_server OPTIONS
(
    filename '/tmp/testdata/data-farms.db721',
    tablename 'Farm'
);

SELECT * FROM db721_farm;