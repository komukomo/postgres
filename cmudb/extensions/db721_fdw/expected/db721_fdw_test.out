CREATE EXTENSION IF NOT EXISTS db721_fdw;
CREATE SERVER IF NOT EXISTS db721_server FOREIGN DATA WRAPPER db721_fdw;
CREATE FOREIGN TABLE IF NOT EXISTS db721_farm
(
    farm_name       varchar,
    min_age_weeks   real,
    max_age_weeks   real
) SERVER db721_server OPTIONS
(
    filename '/home/kapi/git/postgres/data-farms.db721',
    tablename 'Farm'
);
SELECT * FROM db721_farm;
ERROR:  could not devise a query plan for the given query
